pkg/stanza/operator/input/windows/config_windows.go

```diff
@@ -36,7 +36,11 @@ func (c *Config) Build(set component.TelemetrySettings) (operator.Operator, erro
 		return nil, fmt.Errorf("the `start_at` field must be set to `beginning` or `end`")
 	}
 
-	return &Input{
+	if c.Remote.Server != "" && (c.Remote.Username == "" || c.Remote.Password == "") {
```

Comment:
Why is this the only combo we are checking for? Wouldn't it also be invalid if `username` is provided but `server` is not?

I think it's ok to have one error message, but just check that if any one of these fields is populated then all three must be.

```suggestion
	if (c.Remote.Server != "" || c.Remote.Username != "" || c.Remote.Password != "") && // any not empty 
		(c.Remote.Server == "" || c.Remote.Username == "" || c.Remote.Password == "") { // any empty
```